deploy/custom_resources/custom_authorizer/requirements.txt

```diff
@@ -7,4 +7,3 @@ python-jose==3.3.0
 requests==2.31.0
 rsa==4.9
 six==1.16.0
```

Comment:
Just curious, why are we removing this package?

Reply:
I did not see it used anywhere - can keep it and remove and a separate PR not to conflate the 2 though

Reply:
Maybe it is a good idea to keep it separate. We should verify with @TejasRGitHub

Reply:
yup I'll add back

Reply:
all done!